servicetalk-examples/grpc/debugging/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2018 Apple Inc. and the ServiceTalk project authors
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<Configuration status="info">
+  <Appenders>
+    <Console name="Console" target="SYSTEM_OUT">
+      <PatternLayout pattern="%d %30t [%-5level] %-30logger{1} - %msg%n"/>
+    </Console>
+  </Appenders>
+  <Loggers>
+    <!-- Prints server start and shutdown -->
+    <Logger name="io.servicetalk.http.netty.H2ServerParentConnectionContext" level="DEBUG"/>
+
+    <!-- Enables wire logging and h2-frame logging messages -->
+    <Logger name="servicetalk-examples-wire-logger" level="TRACE"/>
+    <Logger name="servicetalk-examples-h2-frame-logger" level="TRACE"/>
+
+    <!-- Prints default subscriber errors-->
+    <Logger name="io.servicetalk.concurrent.api" level="DEBUG"/>
```

Comment:
For this example and for the HTTP one, let's enhance `log4j2.xml` configuration to point (with comments) to some other important loggers that people may be interested in enabling:
1. `io.servicetalk.loadbalancer` - Gives visibility into internal state of the LoadBalancer;
2. `io.servicetalk.dns.discovery.netty` + `io.netty.resolver.dns` - Prints results of each DNS resolution;
3. `io.servicetalk.http` - Gives visibility into HTTP processing;
4. `io.servicetalk.transport` - Gives visibility for transport events.